BUILD.md

```diff
@@ -0,0 +1,31 @@
+## Build Instructions
+
+Disclaimer: These instructions are written by a noob, but it worked to get Grin++ with UI working on Ubuntu 18.04.  Please use at your discretion and edit for other OS's and needs.
+
+### Linux
+#### Prerequisites
+* git
+* build-essential
+* uuid-dev
+* GrinPlusPlus (the node/wallet)
+
+We will be building both GrinPlusPlus and the UI, if you already have GrinPlusPlus built you make have to piecemeal this or build again.
+**Command Line**
+1. ```sudo apt install uuid-dev npm build-essential git```
+2. ```cd ~```
+3. ```git clone https://github.com/GrinPlusPlus/GrinPlusPlus.git```
+4. ```git clone https://github.com/GrinPlusPlus/GrinPlusPlusUI.git```
+5. ```cd GrinPlusPlus```
+6. ```mkdir build```
+7. ```cd build```
+8. ```cmake ..```
+9. ```cmake --build```
```

Comment:
Should be `cmake - - build .`

Reply:
You have it the way I wrote it in the GrinPlusPlus build.md and it worked for me

Reply:
I am wrong, lol.  I'll fix it